pkg/commands/build/status.go

```diff
@@ -27,15 +26,12 @@ import (
 
 const (
 	buildMetadataKey = "io.buildpacks.build.metadata"
-	bomKey           = "bom"
```

Comment:
Is buildMetadataKey used?

Reply:
No, I removed it as part of 831e0496cbc9a6ee8da4e5a997a5487d6c0a8714.